airflow/cli/commands/kubernetes_command.py

```diff
@@ -22,7 +22,7 @@
 from kubernetes.client.api_client import ApiClient
 from kubernetes.client.rest import ApiException
 
-import airflow.utils.yaml as yaml
+from airflow.compat import yaml
```

Comment:
This one I'm not quite sure about -- it's not about compat, but different ways of installing the pyyaml module (with or without C component)